### DISCUSSION TITLE
Why only red and green color?

### DISCUSSION THREAD

**Discussion by Rajgupta7080** [joyceworks/flowchart-react#38] [Q&A]
I want path as blue for success and grey for fail but i am not able to do this.
Please add this feature so user can add the colour by choice.

**Comment by iamppz** [comment]
Okay, I will do it later

获取 Outlook for iOS<https://aka.ms/o0ukef>
________________________________
发件人: RAJ GUPTA ***@***.***>
发送时间: Monday, September 4, 2023 2:24:16 PM
收件人: joyceworks/flowchart-react ***@***.***>
抄送: Subscribed ***@***.***>
主题: [joyceworks/flowchart-react] Why only red and green color? (Discussion #38)


I want path as blue for success and grey for fail but i am not able to do this.
Please add this feature so user can add the colour by choice.

―

**Reply by Rajgupta7080** [reply]
Thank You  @iamppz

**Comment by iamppz** [comment]
Hello, I have released v3.16.1, now you can specify the color of connection by passing a value to its color property:
``` javascript
{
  source: {id: 1, position: "right"},
  destination: {id: 4, position: "left"},
  type: "success",
  title: "1",
  color: "#ff0000"
}
```

**Reply by Rajgupta7080** [reply]
Thank you so much
